amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```
@@ -2091,7 +2094,163 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/events':
+    get:
+      tags:
+        - events
+      summary: List all available retention events
+      description: |
+        Use this to get the list of events that can be used by retention steps
+      operationId: getAllEvents
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventPaging'
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+    post:
+      tags:
+        - events
+      summary: Create a new retention event
+      description: |
+        Create a new event that can be used by retention schedules.
+      operationId: createEvent
+      parameters:
+        - in: body
+          name: eventBodyCreate
+          description: The new event.
+          required: true
+          schema:
+            $ref: '#/definitions/EventBodyCreate'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '201':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventEntry'
+        '400':
+          description: |
+            Invalid parameter: **name**, or **type** or **eventBodyCreate** is invalid
```

```suggestion
            Invalid parameter: **name** or **type** is invalid
```
The eventBodyCreate _is_ the fields name and type.